client/src/components/participant-form/sections/BackgroundInformationSection.js

```diff
@@ -12,6 +12,21 @@ export const BackgroundInformationSection = ({
   isMHAWProgram,
   selectedOption,
 }) => {
+  const [industry, , industryHelpers] = useField('currentOrMostRecentIndustry');
+  const [other, , otherHelpers] = useField('otherIndustry');
+
+  useEffect(() => {
+    if (
+      !other.value &&
+      industry.value &&
+      !currentOrMostRecentIndustryOptions.includes(industry.value)
+    ) {
+      const otherValue = industry.value;
+      industryHelpers.setValue('Other, please specify:');
+      otherHelpers.setValue(otherValue);
+    }
+  }, [industry, other]);
```

Comment:
if we were to fill out the `Other` input and t to submit if the form is invalid, this wipes out the selection for Q14

Before
![Screenshot 2023-10-15 at 8 32 43 PM](https://github.com/bcgov/hcap/assets/99211385/7ee6ea41-6529-4177-8122-6fd455124a3e)

After
![Screenshot 2023-10-15 at 8 32 52 PM](https://github.com/bcgov/hcap/assets/99211385/e2a0de44-fd35-49eb-b301-a3e74e859e72)

Reply:
Good catch! I fixed it.